db/seeds/post_flag_types.yml

```diff
@@ -26,9 +26,17 @@
   active: true
   post_type_id: <%= Question.post_type_id %>
 
+- name: change post status
+  description: >
+    This post should be reopened/closed, locked/unlocked, or deleted/undeleted. Please elaborate in the details field below.

```

Comment:
This wording is showing up correctly in my development environment, so I can select this option when flagging, but unlike the "moderator attention" flag, this one does not insist on the details text field being filled in.

I can see 2 potential solutions:
1. Separate the options sufficiently that each one requires no further details. I guess 3 options would suffice:
    - Open/close
    - Lock/unlock
    - Delete/undelete
2. Keep just one option for all 6 actions, and add an extra property for custom flag types. In addition to "confidential", "active", and "post_type_id", there could also be "requires_details". This would also require adding code elsewhere to make this work.

Option 1 is easy to implement if you consider it sufficient for now. Option 2 seems unnecessary unless you think there will be other cases that need it in future. It doesn't seem worth it just for this particular case, as I'm guessing this flag will be a temporary measure until a more automated approach is implemented (for example, a reopen button available to everyone, that acts similarly to the edit button in that it either takes the action if you have authority, or adds it to a queue if not).

Reply:
Option 3: keep the one flag option, figure out how the code is enforcing the details text for mod flags, and do that.  I see that some flag names are hard-coded, which is something we might need to address in the future, but we could in the meantime do the same thing for this flag type.

Reply:
If option 3 turns out to be easier then I'm fine with that. I'm not worried about hard-coding something that will only be temporary.

*(If anyone reading this is cringing, suspecting that "temporary" may last forever, feel free to raise an issue so it doesn't get forgotten, but I see no reason to fix it in this pull request.)*

Reply:
I decided to hard-code this new curator-attention flag, like we do for the moderator-attention flag.  This code enforces the requirement for the details field, which can't be done through the YAML file currently.  I've removed the reason from the YAML file (and thus we don't need to worry about problems in seeding).